examples/config.yml

```diff
@@ -60,6 +60,9 @@ backplane:
   redisPassword:
   redisNodes:
   jedisPoolMaxTotal: 4000
+  jedisPoolMaxIdle: 4000
+  jedisPoolMinIdle: 500
+  jedisTimeBetweenEvictionRunsMillis: -1
```

Comment:
Could we make these the same as the defaults (from `Backplane.java`?)

Reply:
Fixed at https://github.com/buildfarm/buildfarm/pull/2122/commits/615a8fb0561eb95e7b299d78f8420bd758e47b3f.

Reply:
See them for default
- https://github.com/apache/commons-pool/blob/rel/commons-pool-2.12.0/src/main/java/org/apache/commons/pool2/impl/GenericObjectPoolConfig.java
- https://github.com/redis/jedis/blob/v5.2.0/src/main/java/redis/clients/jedis/ConnectionPoolConfig.java